schemas/iso19139/src/main/plugin/iso19139/layout/tpl-rdf.xsl

```diff
@@ -290,6 +304,10 @@
     </dct:identifier>
     <!-- xpath: gmd:identificationInfo/*/gmd:citation/*/gmd:identifier/*/gmd:code -->
 
+    <!-- SIB addon-->
+    <dcat:landingPage>
+      <xsl:value-of select="$url"/>/srv/api/records/<xsl:value-of select="$uuid"/>
```

Comment:
Might be worth trying to replace `srv` by `<xsl:value-of select="util:getNodeId()"/>`. I believe it should work since https://github.com/geonetwork/core-geonetwork/pull/7191